docs/team/prit3010.md

```diff
@@ -9,4 +9,63 @@ title: Prittam Ravi's Project Portfolio Page
 
 Given below are my contributions to the project.
 
-to be added soon!
+### Summary of Contributions
+
+---
+
+#### Code Contributed
+
+#### Enhancements Implemented
+* **New Features**
+  * Implemented `batch-add` feature
+    * Created a `BatchAddCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/BatchAddCommand.java)) 
+    and `BatchAddCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/BatchAddCommandParser.java))
+    * Use Case: User can add multiple employees at once if the user has them inputted in a `.csv` file
```

Comment:
abit of grammar issue here with the "if the user has them inputted ..." 

Consier "User can add multiple employees at once if they have the data in the form of a `.csv` file" or something to that extent.